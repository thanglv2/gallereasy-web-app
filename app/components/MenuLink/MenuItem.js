import styled from 'styled-components';

const MenuItem = styled.li`
  float: left;
  text-align: center;
  padding: 14px 16px;
  text-decoration: none;
  a {
    text-decoration: none;
    color: ${props => props.theme.grey};
  }
  &.active {
    a {
      color: black;
    }
  }
  @media(max-width: 600px) {
    padding: 15px 10px;
  }
`;

export default MenuItem;