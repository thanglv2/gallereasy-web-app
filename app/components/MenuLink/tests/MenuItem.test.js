import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import MenuItem from '../MenuItem';

describe('<MenuItem />', () => {
  it('render without error', () => {
    const theme = {
      grey: 'grey'
    }
    const tree = renderer.create(<MenuItem theme={theme} />).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
  