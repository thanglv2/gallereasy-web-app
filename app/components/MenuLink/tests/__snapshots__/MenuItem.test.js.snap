// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem /> render without error 1`] = `
.c0 {
  float: left;
  text-align: center;
  padding: 14px 16px;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 a {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: grey;
}

.c0.active a {
  color: black;
}

@media (max-width:600px) {
  .c0 {
    padding: 15px 10px;
  }
}

<li
  className="c0"
/>
`;
